includes/elements/section.php

```diff
@@ -418,16 +418,26 @@ protected function _register_controls() {
 		$this->add_control(
 			'content_position',
 			[
-				'label' => __( 'Content Position', 'elementor' ),
+				'label' => __( 'Vertical Align', 'elementor' ),
 				'type' => Controls_Manager::SELECT,
 				'default' => '',
 				'options' => [
 					'' => __( 'Default', 'elementor' ),
 					'top' => __( 'Top', 'elementor' ),
-					'middle' => __( 'Middle', 'elementor' ),
+					'center' => __( 'Middle', 'elementor' ),
 					'bottom' => __( 'Bottom', 'elementor' ),
-				],
-				'prefix_class' => 'elementor-section-content-',
+                    'space-between' => __( 'Space Between', 'elementor' ),
+                    'space-around' => __( 'Space Around', 'elementor' ),
+                    'space-evenly' => __( 'Space Evenly', 'elementor' ),
+				],
+                'selectors_dictionary' => [
+                    'top' => 'flex-start',
+                    'bottom' => 'flex-end',
+                ],
+                'selectors' => [
+                    '{{WRAPPER}} > .elementor-container > .elementor-row > .elementor-column > .elementor-column-wrap' => 'align-items: {{VALUE}}',
+                    '{{WRAPPER}} > .elementor-container > .elementor-row > .elementor-column > .elementor-column-wrap > .elementor-widget-wrap' => 'align-content: {{VALUE}}',
```

Comment:
Remove this line

Reply:
Vertical positioning does not work properly without this line. 
`flex-start`, `center`, and `flex-end` are `align-item` values, while `space-between`, `space-around` and `space-evenly` are `align-content` values. 
That is why these settings are divided to two separate selectors (also in `column.php`).